modules/account-lib/src/coin/baseCoin/blsKeyPair.ts

```diff
@@ -37,14 +37,28 @@ export abstract class BlsKeyPair implements BaseKeyPair {
    * @param {string} prv a hexadecimal private key
    */
   recordKeysFromPrivateKey(prv: string) {
-    if (this.isValidBLSPrv(prv)) {
+    if (BlsKeyPair.isValidBLSPrv(prv)) {
       const privateKey = BLS.PrivateKey.fromHexString(prv);
       this.keyPair = new BLS.Keypair(privateKey);
     } else {
       throw new Error('Invalid private key');
     }
   }
 
+  /**
+   * Build a keyPair from private key.
+   *
+   * @param {Buffer} prv a UInt8Array private key
+   */
+  recordKeysFromPrivateKeyBytes(prv: Buffer) {
```

Comment:
Same as below. I don't see the need for this duplicity

Reply:
At the very least, if you really want this interface, have one function use the other function to dedupe code